src/s2/util/math/exactfloat/exactfloat.cc

```diff
@@ -71,7 +71,7 @@ inline static void BN_ext_set_uint64(BIGNUM* bn, uint64 v) {
 // Return the absolute value of a BIGNUM as a 64-bit unsigned integer.
 // Requires that BIGNUM fits into 64 bits.
 inline static uint64 BN_ext_get_uint64(const BIGNUM* bn) {
-  uint64 u64;
+  std::uint64_t u64;
```

Comment:
just use `uint64_t`. it should work everywhere.

Reply:
Done.